modules/oc-mirror-updating-registry-about.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="oc-mirror-updating-registry-about_{context}"]
+= About updating your mirror registry content
+
+When you run the oc-mirror plug-in again, it generates an image set that only contains new and updated images since the previous execution. Because it only pulls in the differences since the previous image set was created, the generated image set is often smaller and faster to process than the initial image set.
+
+[IMPORTANT]
+====
+Generated image sets are sequential and must be pushed to the target mirror registry in order. You can derive the sequence number from the file name of the generated image set archive file.
+====
+
+[discrete]
+== Adding new and updated images
+
+Depending on the settings in your image set configuration, future executions of oc-mirror can mirror additional new and updated images. Review the settings in your image set configuration to ensure that you are retrieving new versions as necessary. For example, you can set the minimum and maximum versions of Operators to mirror if you want to restrict to specific versions. Or you can opt to retrieve the full Operator catalog, so that you always receive the latest versions during future oc-mirror executions.
+
+[discrete]
+== Pruning images
+
+Images are pruned automatically from the target mirror registry if they are no longer included in the latest image set that was generated and mirrored. If there are {product-title} releases or Operator versions that you no longer need, you can modify your image set configuration to exclude them, and they will be pruned from the mirror registry upon mirroring. This allows you to easily manage and clean up unneeded content and reclaim storage resources.
```

Comment:
```suggestion
Images are pruned automatically from the target mirror registry if they are no longer included in the latest image set that was generated and mirrored. If there are {product-title} releases or Operator versions that you no longer need, you can modify your image set configuration to exclude them, and they will be pruned from the mirror registry upon mirroring. This can be done by adjusting a minimum / maximum version range setting per operator in the image set configuration file or by deleting the operator from the list of operators to be mirrored from the catalog. It also works when you remove entire operator catalogs or entire OpenShift releases from the configuration file. This allows you to easily manage and clean up unneeded content and reclaim storage resources.
```